tests/failing-ui-tests.txt

```diff
@@ -75,4 +75,4 @@ tests/ui/async-await/in-trait/dont-project-to-specializable-projection.rs
 tests/ui/consts/try-operator.rs
 tests/ui/coroutine/unwind-abort-mix.rs
 tests/ui/type-alias-impl-trait/rpit_tait_equality_in_canonical_query.rs
-tests/ui/impl-trait/equality-in-canonical-query.rs
+tests/ui/impl-trait/equality-in-canonical-query.rs
```

Comment:
The error is caused by this change: it seems your text editor removed the newline at the end, so please revert this change.

Reply:
This one space wasted one day 😭😭